plugins/rbac/src/components/CreateRole/AddMembersForm.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+
+import { stringifyEntityRef } from '@backstage/catalog-model';
+
+import { Box, LinearProgress, makeStyles, TextField } from '@material-ui/core';
```

Comment:
Would it be worth favoring material 5 components at this stage?  Or we can wait until it's mandatory at some point in the future.

Reply:
This is part of Divyanshi's Create Role PR. What do you suggest @divyanshiGupta & @parvathyvr  ?